### ISSUE TITLE
Suggestion: Use PHP builtin methods for IP lookups

### ISSUE THREAD

**Issue by Brueggus** [cp6/my-idlers#7]
Instead of using an external API, there's a builtin method to do DNS lookups instead:
https://www.php.net/manual/de/function.dns-get-record.php

**Comment by cp6** [comment]
This was returning very minimal data from my testing:

`Array ( [0] => Array ( [host] => removedhost.com [class] => IN [ttl] => 3788 [type] => HINFO [cpu] => RFC8482 [os] => ) ) `

**Comment by Brueggus** [comment]
Interesting... did you really use `removedhost.com` for testing. That one has no A/AAAA records.

Could you try the following snippet?

```
<pre>
<?php
$host = "mirror.netcologne.de";

$result = dns_get_record($host, DNS_A);
print_r($result);

$result = dns_get_record($host, DNS_AAAA);
print_r($result);
?>
</pre>
```

**Comment by cp6** [comment]
Ok, no I removed it for privacy. I see now that you have to define what you want and not let it run the default type. I will look at it or feel free to do a commit :)

**Comment by asimzeeshan** [comment]
@cp6 I have applied a fix in a89f624b082a81a7dc4d8deef7c33964a849b40c
needs to be tested, available in my pull request
